pkg/controller/applicationlayer/applicationlayer_controller.go

```diff
@@ -304,16 +304,25 @@ func (r *ReconcileApplicationLayer) Reconcile(ctx context.Context, request recon
 
 // updateApplicationLayerWithDefaults populates the applicationlayer with defaults.
 func updateApplicationLayerWithDefaults(al *operatorv1.ApplicationLayer) {
-	defaultLogIntervalSeconds := int64(5)
-	defaultLogRequestsPerInterval := int64(-1)
+	var defaultLogIntervalSeconds int64 = 5
+	var defaultLogRequestsPerInterval int64 = -1
```

Comment:
I know it was this way before your PR, but why are we defaulting this to -1? What does a value of -1 mean for a rate field? Seems like it would have no semantic meaning to me - we should probably disallow negative numbers for this field and other time based fields, potentially even using `uint` here instead.

Reply:
I have seen that Suresh replied to this question in the operator-dev slack channel. Just to keep the record of this information here's a snippet from CRD:
```'Maximum number of unique L7 logs that are sent LogIntervalSeconds.
                      Adjust this to limit the number of L7 logs sent per LogIntervalSeconds
                      to felix for further processing, use negative number to ignore
                      limits. Default: -1'